src/views/virtualmachines/details/tabs/configuration/VirtualMachineConfigurationTab.tsx

```diff
@@ -16,9 +22,12 @@ import { getInnerTabFromPath, includesConfigurationPath, tabs } from './utils/ut
 
 import './virtual-machine-configuration-tab.scss';
 
-const VirtualMachineConfigurationTab: FC<NavPageComponentProps> = ({ vm }) => {
+const VirtualMachineConfigurationTab: FC<NavPageComponentProps> = ({ vm: obj }) => {
   const navigate = useNavigate();
   const location = useLocation();
+  useSignals();
```

Comment:
we need it here? guide tour isn't enough?

Reply:
removed